packages/button/README.md

```diff
@@ -0,0 +1,56 @@
+# Buttons
+
+Buttons are used for action items. Some examples include Create, Save, Delete. Buttons make it obvious to the user that they can take an action.
+
+Do not use buttons as navigational elements. Instead, use anchors because it takes the user to a new page and is not associated with an action.
+
+Each page will likely have one primary button. Any remaining calls-to-action are represented as secondary buttons.
+
+## Button states and types
+
+### Types
+
+**Primary**: Primary call to action button. Should stand out above all other actions. Do not put two primary buttons next to each other. Examples include Save, Deploy.
+
+**Secondary**: Most buttons on a page are likely this style.
+
+**Danger**: Destructive buttons are used to highlight something dangerous that can't be undone e.g. "Delete".
+
+**Icon only**: Use when an icon is enough to represent the action e.g. "Create a Service". Should be accompanied by a tooltip.
+
+**Icon with label**: If there is space and we feel an icon by itself is not obvious enough e.g. "Kill Task"
+
+### States
+
+**Disabled**: Disable a button when the action is not applicable but we want to show the user that it may be possible. Disabled buttons should have a tooltip.
```

Comment:
Should ButtonBase implement this recommendation through a disabledHelpText prop or something?